CMakeLists.txt

```diff
@@ -507,7 +507,7 @@ install(
   DESTINATION cmake)
 
 install(
-  DIRECTORY etc DESTINATION  ${CMAKE_INSTALL_DATAROOTDIR}/foedag/etc/)
+  DIRECTORY etc/ DESTINATION  ${CMAKE_INSTALL_DATAROOTDIR}/foedag/etc/)
```

Comment:
Conflict here with main.

Reply:
Resolved, the earlier change to main accomplished the same thing so using that change. Verified it still works w/ `sudo make uninstall` followed by `sudo make install` and checking for duplicate etc folders in /usr/local.